ext/OceananigansNCDatasetsExt/OceananigansNCDatasetsExt.jl

```diff
@@ -0,0 +1,84 @@
+module OceananigansNCDatasetsExt
+
+using NCDatasets
+
+using Dates: AbstractTime, UTC, now, DateTime
+using Printf: @sprintf
+using OrderedCollections: OrderedDict
+using SeawaterPolynomials: BoussinesqEquationOfState
+using Statistics: mean
+
+using Oceananigans: initialize!, prettytime, pretty_filesize, AbstractModel
+using Oceananigans.Architectures: CPU, GPU, on_architecture
+using Oceananigans.AbstractOperations: KernelFunctionOperation, AbstractOperation
+using Oceananigans.BuoyancyFormulations: BuoyancyForce, BuoyancyTracer, SeawaterBuoyancy, LinearEquationOfState
+using Oceananigans.Fields
+using Oceananigans.Fields: Reduction, reduced_dimensions, reduced_location, location, indices
+using Oceananigans.Grids: Center, Face, Flat, Periodic, Bounded,
+                          AbstractGrid, RectilinearGrid, LatitudeLongitudeGrid, StaticVerticalDiscretization,
+                          topology, halo_size, xspacings, yspacings, zspacings, λspacings, φspacings,
+                          parent_index_range, nodes, ξnodes, ηnodes, rnodes, validate_index, peripheral_node,
+                          constructor_arguments, architecture
+using Oceananigans.ImmersedBoundaries: ImmersedBoundaryGrid, GridFittedBottom, GFBIBG, GridFittedBoundary, PartialCellBottom, PCBIBG,
+                                       CenterImmersedCondition, InterfaceImmersedCondition
+using Oceananigans.Models: ShallowWaterModel, LagrangianParticles
+using Oceananigans.Utils: TimeInterval, IterationInterval, WallTimeInterval, materialize_schedule,
+                          versioninfo_with_gpu, oceananigans_versioninfo, prettykeys
+using Oceananigans.OutputWriters:
+    auto_extension,
+    output_averaging_schedule,
+    show_averaging_schedule,
+    AveragedTimeInterval,
+    WindowedTimeAverage,
+    NoFileSplitting,
+    update_file_splitting_schedule!,
+    construct_output,
+    time_average_outputs,
+    restrict_to_interior,
+    fetch_output,
+    convert_output,
+    fetch_and_convert_output,
+    show_array_type
+using Oceananigans.OutputReaders: InMemoryFTS, time_indices
```

Comment:
```suggestion
    show_array_type
    
using Oceananigans.OutputReaders: InMemoryFTS, time_indices
```

totally trivial comment but can you add some whitespace to this file? I think it makes it more readable. I think its best to have whitespace following multiline code blocks .